recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3.16.4" %}
+{% set version = "3.17.0" %}
 
 package:
   name: cmake
   version: {{ version }}
 
 source:
   url: https://gitlab.kitware.com/cmake/cmake/-/archive/v{{ version }}/cmake-v{{ version }}.tar.bz2
-  sha256: 40e0dec6dc9e36820e001b8425aa4328a0b42f1915b14d68aee116e25c3d34df
+  sha256: 9791a8ff223ffbecc5a8985d4a623be8bff037881356f7147fa616e18c23f8c6
   patches:
     - patches/3.16.2/0001-find_-Add-debug-logging-infrastructure.patch
     - patches/3.16.2/0002-find_-Use-debug-logging-infrastructure.patch
```

Comment:
We may wind up needing to drop or alter these patches, but let's wait for CI to report.

Reply:
@mingwandroid, appears you added these, do you know if we still need them for 3.17.0?

Reply:
Yeah looks like these don't apply on CI. Probably got fixed upstream. Let's try dropping them 🙂